tests/test_error.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from milabench.testing import milabench_cmd, config
+
+
+expected_output = """"""
+
+
+def test_error_reporting(capsys):
+
+    with pytest.raises(SystemExit) as err:
+        milabench_cmd("run", config("oom"))
+
+    assert err.type is SystemExit
+    assert err.value.code == -1
+
+    captured = capsys.readouterr()
+    assert captured.out == expected_output
```

Comment:
Add a check on `captured.err` as well, checking that a distinctive part of the expected error output is there is enough.

Reply:
I added a check and stderr is empty in that case